text/0000-project-groups.md

```diff
@@ -0,0 +1,391 @@
+  status of the group. A group can be "restored" at a later stage.
+- Reasons to archive:
+  - Nobody in the group has time anymore or higher priority things arose.
+  - There's a blocking issue that can't be resolved.
+  - Don't see any additional work to do in this area in the near future.
+  - The work was done to a satisfactory state.
+  - The group decided the idea wasn't so good after all.
+
+7. Create a blog post announcing the archival of the group.
+
+- The scope of this post will vary based on the scope of the group, but
+  ideally it would include some of the following.
+  - Overview of decisions, RFCs, and other output the group produced.
+  - Thoughts on the process, how it worked (or didn't as case may be), any
+    difficulties encountered, and what they would want to be improved.
+
+8. Archive infrastructure.
+
+- Archive GitHub repository to be read-only.
+- Archive chat channel(s) on any platforms.
+
+9. (Optional) Restore group
+
+- At any later point the group could be restored to active status if there are
+  assigned liaisons and shepherds, and the group has consensus from the team
+  that the group should become active again.
+- If significant time has passed, part of restoring the group should be to
+  evaluate whether the past decisions and rationale are still applicable to the
+  present.
+- If there is consensus to become active again, go to step 3.
+
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[internals]: https://internals.rust-lang.org
+
+# Reference-level explanation
+
+## Common Aspects of Working Groups and Project Groups
+
+Before going into the differences between these groups it's important to start
+with what it is shared between them.
+
+- A charter that defines the scope and intent of the group.
+
+- A GitHub repository hosted under the `rust-lang` organization
+  (or separate organisation if necessary) containing the charter and
+  instructions for how community members can monitor or participate in
+  the group.
+
+- Groups have at least one shepherd who acts as the leader of the group and is
+  typically responsible for writing the initial charter, handling administrative
+  and communication tasks, as well as delegating responsibilities to other
+  members in the group.
+
+- Groups should have a liaison member associated with an official Rust team.
+
+  - This liaison may also be but is not required to be a shepherd.
+
+- Groups and their members are represented on the official
+  [rust-lang.org] website.
+
+- Membership requirements for both groups is decided by the shepherd and should
+  be stated in the charter.
+
+  - Initial membership should try to represent people who have already been
+    participating regularly and productively in the respective area.
+
+- Neither group has _"formal decision making power"_: meaning that they are not
+  able to accept RFCs on `rust-lang/rfcs`. Similarly, neither group has
+  representation on the Core team.
```

Comment:
One thing that should probably be explicitly mentioned: core team members are not excluded from joining groups.

Reply:
We've never had exclusionary conditions on joining before before so I think this is implied.